plugins/header_rewrite/regex_helper.h

```diff
@@ -36,11 +36,13 @@ class regexHelper
   regexHelper() : regex(NULL), regexExtra(NULL), regexCcount(0) {}
   ~regexHelper()
   {
-    if (regex)
+    if (regex) {
       pcre_free(regex);
+    }
 
-    if (regexExtra)
+    if (regexExtra) {
       pcre_free(regexExtra);
+    }
```

Comment:
FWIW, ``pcre_free`` should have the same semantics as ``free(3)`` so passing ``NULL`` is a no-op.